src/Symfony/Bundle/DebugBundle/DependencyInjection/Configuration.php

```diff
@@ -51,6 +51,15 @@ public function getConfigTreeBuilder()
                     ->example('php://stderr')
                     ->defaultNull()
                 ->end()
+                ->arrayNode('server_dump')->canBeEnabled()
+                    ->children()
+                        ->scalarNode('host')->defaultValue('tcp://0.0.0.0:9912')->end()
```

Comment:
I would not make this configurable in the config but as an env var. Like for the web server, imagine a scenario where you are working on many projects in parallel. It's easier to tweak an env var from the outside. Also, if the server is not a Symfony app, having the server address as an env var helps as well :)